# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xine-ui/hotfix-getopt.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Fix the build with C23: always use the full getopt() prototype.

--- xine-ui-0.99.14.orig/src/common/getopt.h	2023-01-07 20:28:52.000000000 +0100
+++ xine-ui-0.99.14/src/common/getopt.h	2026-09-27 18:34:35.691682486 +0200
@@ -133,7 +133,7 @@ struct option
    errors, only prototype getopt for the GNU C library.  */
 extern int getopt (int __argc, char *const *__argv, const char *__shortopts);
 # else /* not __GNU_LIBRARY__ */
-extern int getopt ();
+extern int getopt (int __argc, char *const *__argv, const char *__shortopts);
 # endif /* __GNU_LIBRARY__ */
 
 # ifndef __need_getopt
